### ISSUE TITLE
Test alembic migrations

### ISSUE THREAD

**Issue by jwhitlock** [mozilla-it/ctms-api#68]
Integrate [pytest-alembic](https://github.com/schireson/pytest-alembic) (ht @bsieber-mozilla) to test these scenarios:

* Are the models and migrations in sync?
* Is there a single alembic head? (can be broken if two database PRs were merged)
* (Future) Migrate up to a revision, add some problematic production-like data, run the remaining migrations

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/CTMS-50)

**Comment by jwhitlock** [comment]
Oops, did not mean to close this one

**Comment by jwhitlock** [comment]
I spent some time on this, but didn't quite get it working. My code is in a branch in my fork, https://github.com/mozilla-it/ctms-api/compare/main...jwhitlock:wip-test-alembic-migrations. I think I could get it working with a few more hours, but it isn't critical for the current milestone.

I was getting pylint errors in my editor, and I spent some time getting rid of these. Some of these could be extracted to a new PR.
